core/themes/basis/css/skin.css

```diff
@@ -1226,3 +1226,48 @@ div.error:before {
   /* background-image: -webkit-linear-gradient(transparent, transparent), url(../../../misc/message-error.svg); */
   /* background-image: linear-gradient(transparent, transparent), url(../../../misc/message-error.svg); */
 }
+
+/* Card view on default home page */
```

Comment:
@stpaultim you may not be aware, but this section in skin.css is inside a `/* Color Module: Don't touch */` section, which means that color changes will not get processed here.

If that's by intention, it's OK, but if it just was an oversight, you might want to move that CSS above the "Don't touch" in line 1162.

https://github.com/backdrop/backdrop/blob/1.x/core/modules/color/color.module#L669